integration/token/fungible/topology/topology.go

```diff
@@ -360,6 +367,22 @@ func Topology(opts common.Opts) []api.Topology {
 	for _, sdk := range opts.SDKs {
 		fscTopology.AddSDK(sdk)
 	}
+
+	// any extra TMS
+	for _, tmsOpts := range opts.ExtraTMSs {
+		tms := tokenTopology.AddTMS(nodeList, backendNetwork, backendChannel, tmsOpts.TokenSDKDriver)
+		tms.Alias = tmsOpts.Alias
+		tms.Namespace = "token-chaincode"
+		tms.Transient = true
```

Comment:
What does transient stand for?

Reply:
I'll add a comment, but it means that it is not an actual TMS, it is a TMS for which we need to generate only the public parameters